README.md

```diff
@@ -35,42 +59,47 @@ on:
     branches: [master]
 jobs:
   upload-to-mobile-dev:
-    name: Run Flows on Maestro Cloud
+    name: Run Flows on Robin
     steps:
       - uses: actions/checkout@v3
         with:
           ref: ${{ github.event.pull_request.head.sha }} # Checkout PR HEAD
 ```
 
-# Android
+For more information on triggering workflows, check out [GitHub's documentation](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows).
+
+## Android
 
 ```yaml
 - uses: mobile-dev-inc/action-maestro-cloud@v1.9.4
   with:
-    api-key: ${{ secrets.MAESTRO_CLOUD_API_KEY }}
+    api-key: ${{ secrets.ROBIN_API_KEY }}
+    project-id: 'proj_01abc3def456ghij78klmnopq9'
```

Comment:
@Fishbowler I guess pasting in the project id here wasn't intended? It's not secret, so we could in theory have real project id here but I propose we replace it with something like `<your_project_id>` 🙂

Reply:
It's a fake project ID, and our docs make it clear it's not a secret, so I wanted to include an example of the string format.

Chances are, folk will include the action once in a project - they're not going to abstract the value to a GH secret.